tests/test_dev_server.py

```diff
@@ -2,38 +2,37 @@
 import io
 import socket
 from socketserver import BaseServer
-from unittest import mock
+from unittest.mock import patch, MagicMock, call
 
-from lbz.response import Response
-from lbz.resource import Resource
-from lbz.router import add_route
+import pytest
 
-from lbz.dev.server import MyDevServer
-from lbz.dev.server import MyLambdaDevHandler
+from lbz.dev.server import MyDevServer, MyLambdaDevHandler
+from tests.sample_test_resources import SimpleTestResource
 
+# TODO: Check why in some tests there are two responses - its not affecting actual runtime
 
-class HelloWorld(Resource):
-    @add_route("/", method="GET")
-    def list(self):
-        return Response({"message": "HelloWorld"})
 
-    @add_route("/t/{id}", method="GET")
-    def get(self):
-        return Response({"message": "HelloWorld"})
+def test_my_dev_server_no_cls_raises() -> None:
+    with pytest.raises(TypeError):
+        MyLambdaDevHandler.cls()  # pylint: disable=no-value-for-parameter
```

Comment:
That is a totally wrong usage of `cls` property 🤔  I don't see value from checking that this way. Can you explain it?